internal/reporter/slack/reporter.go

```diff
@@ -216,27 +238,37 @@ func convertRPCImageListToK8SImageList(images []*rpc.Image) []s2hv1.Image {
 	return k8sImages
 }
 
-func (r *reporter) makeComponentUpgradeReport(comp *internal.ComponentUpgradeReporter) string {
+func (r *reporter) makeComponentUpgradeReport(comp *internal.ComponentUpgradeReporter, extraMessage string) string {
 	queueHistURL := `{{ .SamsahaiExternalURL }}/teams/{{ .TeamName }}/queue/histories/{{ .QueueHistoryName }}`
 	queueLogURL := `{{ .SamsahaiExternalURL }}/teams/{{ .TeamName }}/queue/histories/{{ .QueueHistoryName }}/log`
 
+	var extraMessageReport string
+	if extraMessage != "" {
+		extraMessageReport = fmt.Sprintf("*Message:* %s", extraMessage)
+	}
+
 	message := `
 *Component Upgrade:* {{ .StatusStr }}
-` + r.makeDeploymentQueueReport(comp, queueHistURL, queueLogURL)
+` + r.makeDeploymentQueueReport(comp, queueHistURL, queueLogURL) + "\n" + extraMessageReport
```

Comment:
maybe could we add new line (`\n`) in `extraMessageReport` instead, before the message